### ISSUE TITLE
How to use Login as a API

### ISSUE THREAD

**Issue by FazilaMehtab** [LaravelDaily/Laravel-Passport-API-Server-Client-Demo#3]
Hi,
Currently server has its own login form but I want to use login from postman as a API because I have my login in my application and I don't want keep my loginform on server . Is it possible in it if yes please help me how to do that.

Please respond me ASAP.

Thanks

**Comment by LaravelDaily** [comment]
Yes it is possible but it is outside of this demo project, check the Laravel Passport documentation for this, or look for tutorials online.
